apps/ledger-live-desktop/src/renderer/families/ethereum/StakeFlowModal/index.tsx

```diff
@@ -11,8 +11,9 @@ type Props = {
   account: Account;
   singleProviderRedirectMode?: boolean;
   source?: string;
+  checkbox?: boolean;
 };
-const DelegationModal = ({ account, singleProviderRedirectMode, source }: Props) => {
+const DelegationModal = ({ account, checkbox, singleProviderRedirectMode, source }: Props) => {
```

Comment:
change the props name "checkbox" into "hasCheckbox" would make more sense.

Reply:
you are totally right, changed!